src/domain/fiat/display-currency.ts

```diff
@@ -23,3 +23,14 @@ export const DisplayCurrencyConverter = (
     return Math.floor(Number(amount) / (price * CENTS_PER_USD)) as Satoshis
   },
 })
+
+export const NewDisplayCurrencyConverter = (
+  displayCurrencyPrice: DisplayCurrencyBasePerSat,
+): NewDisplayCurrencyConverter => {
+  return {
+    fromBtcAmount: (btc: BtcPaymentAmount): DisplayCurrencyBaseAmount =>
+      (Number(btc.amount) * displayCurrencyPrice) as DisplayCurrencyBaseAmount,
+    fromUsdAmount: (usd: UsdPaymentAmount): DisplayCurrencyBaseAmount =>
+      Number(usd.amount) as DisplayCurrencyBaseAmount,
```

Comment:
Should we throw some kind of Error here if the DisplayCurrency isn't USD?

Reply:
I was thinking about this as well. I think the correct fix though is to have the price service return prices for 2 bases as @dolcalmi intends (`displayCurrencyPerSat` & `displayCurrencyPerCent`) and to then use that 2nd price to correctly implement this function.

I can probably apply that interface here in an incomplete form in the meantime 🤔